tests/tdastro/sources/test_snia.py

```diff
@@ -174,22 +154,22 @@ def test_snia_end2end(
         saltpars = {"x0": p["x0"], "x1": p["x1"], "c": p["c"], "z": p["redshift"], "t0": p["t0"]}
         model = sncosmo.Model(sncosmo_modelname)
         model.update(saltpars)
-        z = p["redshift"]
         wave = passbands.waves
-        if opsim is None:
-            time = phase_rest * (1 + z) + p["t0"]
-            assert np.allclose(res["times"], time)
-        else:
-            time = res["times"]
+        time = res["times"]
 
         flux_sncosmo = model.flux(time, wave)
-        assert np.allclose(res["flux_flam"] * 1e10, flux_sncosmo * 1e10)
+        np.testing.assert_allclose(res["flux_flam"], flux_sncosmo, atol=1e-30, rtol=1e-5)
 
-        for f in passbands.passbands:
-            bandflux_sncosmo = model.bandflux(f.replace("_", ""), time, zpsys="ab", zp=8.9 + 2.5 * 9)
-            assert np.allclose(res["bandfluxes"][f], bandflux_sncosmo, rtol=0.1)
+        for f, passband in passbands.passbands.items():
+            # Skip test for negative fluxes
```

Comment:
I guess it's fine to do this for now, but would like to revisit it in the future.

Reply:
I've created #133 for that